modal.css

```diff
@@ -0,0 +1,126 @@
+/* Styling for the modal */
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  padding: 0 10px;
+}
```

Comment:
- [ ] Kindly fix the `scrolling` behavior of your modal. For example, in the mobile, I have a design similar to the screenshot below 👇🏽  The `live version` and `source code` buttons are not seen 👀 The issue is when I scroll down to access them, only the `body` element is scrolling.

![Screenshot from 2023-07-29 11-37-29](https://github.com/DevKaranJ/DevPortfolio/assets/10439283/185c0a35-d824-4acb-914e-c2ad054288d0)